src/features/landing/components/ForWhoSection/ForWhoSection.tsx

```
@@ -1,27 +1,43 @@
 
 function ForWhoSection(): React.ReactElement {
-  const iconAltText: string = 'Vector';
+  const { t } = useTranslation();
 
-  const tabletProps: ImageProps = getOptimizedImageProps({
-    src: TabletVectorIcon,
-    alt: iconAltText,
+  const bigScreenProps = getOptimizedImageProps({
+    src: bigScreen,
+    alt: t('alts.bigScreen')
   }).props;
```

ESLint

[error] Expected bigScreenProps to have a type annotation.

(@typescript-eslint/typedef)